backend/typescript/services/implementations/camperService.ts

```diff
@@ -363,6 +365,7 @@ class CamperService implements ICamperService {
       contactEmail: waitlistedCamper.contactEmail,
       contactNumber: waitlistedCamper.contactNumber,
       campSession: waitlistedCamper.campSession,
+      status: "NOT REGISTERED",
```

Comment:
let's just change this to use waitlistedCamper to reduce amount of hard coding
```suggestion
      status: waitlistedCamper.status,
```